usecases/decision_usecase.go

```diff
@@ -600,7 +600,7 @@ func (usecase *DecisionUsecase) CreateAllDecisions(
 				Observe(time.Since(decisionStart).Seconds())
 
 			for _, sce := range item.decision.ScreeningExecutions {
-				err := usecase.screeningRepository.InsertScreening(ctx, tx, sce)
+				err = usecase.screeningRepository.InsertScreening(ctx, tx, sce)
```

Comment:
Is that an effective functional change? Because from what I see, it doesn't change anything (it is also redefined in an inner scope and after the current scope.